src/styles/header.scss

```diff
@@ -0,0 +1,64 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
+  background-color: #fff;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+
+    list-style: none;
+  }
+
+  &__item {
+    width: fit-content;
+    margin-right: 20px;
```

Comment:
It's a good practice to use variable for margins and paddings. It will make your code more maintainable and consistent.